footprints/main/management/commands/update_imprints.py

```diff
@@ -15,21 +39,105 @@ def csvfile_reader(self, filename):
         csv_file = open(filename)
```

Comment:
This is rather nit-picky because it's a management command, but this leaves the file pointer dangling. It might be worth turning this into a context manager, or just use `with open('foo') as fp:` below.

Reply:
I'll add an explicit close.